Wireframe/index.html

```diff
@@ -8,21 +8,40 @@
   </head>
   <body>
     <header>
-      <h1>Wireframe</h1>
-      <p>
-        This is the default, provided code and no changes have been made yet.
-      </p>
+       <h1>Version control</h1>
+        <p>
+          The purposes of README file, wireframe and Git branch
+        </p>
     </header>
     <main>
+
+      <article>
+        <img src="placeholder.svg" alt="Slategrey triangle imposing over a dark slategrey triangle" />
```

Comment:
I think the idea is to replace any image like 'placeholder'. I can suggest using [unsplash](unsplash.com) for free images.